plugins/auth-backend/src/providers/oauth2/provider.ts

```diff
@@ -153,7 +153,7 @@ export function createOAuth2Provider(
   const appOrigin = envConfig.getString('appOrigin');
   const clientID = envConfig.getString('clientId');
   const clientSecret = envConfig.getString('clientSecret');
-  const callbackURL = `${baseUrl}/${providerId}/handler/frame?env=${env}`;
```

Comment:
Realized this only removes the param from okta, I think we can remove it from all the OAuth providers?

Reply:
I didn't notice it either. Yes, will remove the env parameter from the callbackURL of other providers also.